### PR TITLE
Added rel 5.7.08 of camlp5.

### PR DESCRIPTION
**Pull request by bschommer**
The latest release of camlp5 which should be compatible with OCaml 4.08.

### CODE REVIEW

File: packages/camlp5/camlp5.7.08/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+license: "BSD-3-Clause"
+build: [
+  ["./configure" "--prefix" prefix "-libdir" lib "-mandir" man]
+  [make "world.opt"]
+]
+bug-reports: "https://github.com/camlp5/camlp5/issues"
+dev-repo: "git+https://github.com/camlp5/camlp5.git"
+doc: "https://camlp5.github.io/doc/html"
+install: [make "install"]
+remove: [
```
Comment (kit-ty-kate) [inline packages/camlp5/camlp5.7.08/opam]: The remove field is unnecessary since opam 2.0

**Comment by kit-ty-kate** [comment]
Good to go once the remove field has been removed

**Comment by bschommer** [comment]
The remove field is gone.

**Comment by kit-ty-kate** [comment]
thanks a lot!

**Pull request merged**